web-client/src/presenter/computeds/Public/publicTrialSessionDetailsHelper.test.ts

```diff
@@ -13,8 +13,12 @@ describe('publicTrialSessionDetailsHelper', () => {
 
   const MOCK_SEALED_CASE = cloneDeep(MOCK_CASE);
   MOCK_SEALED_CASE.isSealed = true;
+  MOCK_SEALED_CASE.docketNumber = '101-23';
+  MOCK_SEALED_CASE.docketNumberWithSuffix = '101-23';
 
   const MOCK_LEAD_CASE = cloneDeep(MOCK_CASE);
+  MOCK_LEAD_CASE.docketNumber = '100-23';
+  MOCK_LEAD_CASE.docketNumberWithSuffix = MOCK_LEAD_CASE.docketNumber;
   MOCK_LEAD_CASE.leadDocketNumber = MOCK_LEAD_CASE.docketNumber;
```

Comment:
Should one of these have a suffix?

Reply:
`MOCK_CONSOLIDATED_CASE` below does :) These don't need one.